fonts.mk

```diff
@@ -0,0 +1,521 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS    += Fonts
```

Comment:
Should be `x11-fonts` instead of `Fonts` (and `x11-fonts.mk` instead of `fonts.mk`)